src/components/header/header.scss

```diff
@@ -0,0 +1,60 @@
+$primary-color: rgb(27, 41, 48);
+$logo-color: rgb(246, 205, 77);
+$border-color: rgb(150, 150, 150);
+$social-icon-size: 4rem;
+
+#header {
+  background: $primary-color;
+  display: flex;
+  height: 10rem;
+  color: #fff;
+}
+
+.logo {
+  margin: 1rem 0 1rem 0;
+}
+
+.logo path {
+  fill: $logo-color;
+}
+
+.logo svg {
+  padding: .75rem 0 .75rem 0;
+  height: 100%;
+}
+
+.social {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.social div {
+  margin-right: $social-icon-size/1.5;
+}
+
+.fa.fa-facebook {
+  color: #fff;
```

Comment:
IMHO, I'd store all colors as variables. If by chance you decide true white is _too_ white, it's so much easier to update the variable once. Same with social media colors.